frontend/src/components/card/Card.tsx

```diff
@@ -23,6 +23,7 @@ const Card: FunctionComponent<CardProps> = ({
   return (
     <div
       className={`${classes.root} ${classes[variant]} ${
+        // TODO: fix the possibly undefined index "variant"
```

Comment:
The above improvement would also solve this issue of having to check whether the provided variant is even implemented in the scss styles.